src/custom-elements/src/components/tabs/tabs.tsx

```diff
@@ -159,8 +159,10 @@ export class Tabs {
   }
 
   activatePanel(panelId?: string): void {
+    if (!panelId) return;
+
     this.el.querySelector(`.${TABS_CLASSES.PANEL}.${ACTIVE_CLASS}`)?.classList.remove(ACTIVE_CLASS);
-    this.el.querySelector(`.${TABS_CLASSES.PANEL}#${panelId}`).classList.add(ACTIVE_CLASS);
+    this.el.querySelector(`.${TABS_CLASSES.PANEL}#${panelId}`)?.classList.add(ACTIVE_CLASS);
```

Comment:
Is the `?` necessary? 

I wonder if the situation where the active panel is deactivated but there is not a new panel being activated could happen.